DQM/CastorMonitor/src/CastorRecHitMonitor.cc

```diff
@@ -192,24 +196,26 @@ void CastorRecHitMonitor::processEvent(const CastorRecHitCollection& castorHits)
     double es = 0.;
     for (int z=0; z<14; z++) {
       float rh = energyInEachChannel[z][phi]*0.001;
-      int ind = phi*14 + z +1;
+      int ind = z*16 + phi +1;
+//    int ind = phi*14 + z +1;
       h2RHchan->Fill(ind,rh);
       hallchan->Fill(rh);
       if(rh < 0.) continue;      
-      h2RHmap->Fill(z,phi,rh); 
+//      h2RHmap->Fill(z,phi,rh);
+      h2RHoccmap->Fill(z,phi,rh);
       es += rh;
     }
     h2RHvsSec->Fill(phi,es);
     etot += es;
   } // end for(int phi=0;
-
+/*
   if(ievt_ %100 == 0) 
     for(int mod=1; mod<=14; mod++) 
       for(int sec=1; sec<=16;sec++) {
 	double a= h2RHmap->getTH2F()->GetBinContent(mod,sec);
 	h2RHoccmap->getTH2F()->SetBinContent(mod,sec,a/double(ievt_));
       }
-
+*/
```

Comment:
Are you sure that this block of commented out code is necessary here?

Reply:
cleaned commented out lines:
CastorRecHitMonitor.cc
CastorDigiMonitor.cc, CastorDigiMonitor.h
CastorMonitorModule.cc